scripts/install_ebpf.psm1

```diff
@@ -122,7 +177,8 @@ function Start-eBPFComponents
 
 function Install-eBPFComponents
 {
-    param([parameter(Mandatory=$false)] [bool] $Tracing = $false,
+    param([parameter(Mandatory=$false)] [bool] $KmTracing = $false,
+          [parameter(Mandatory=$false)] [string] $KmTraceType = "file",
```

Comment:
since KMTraceType is optional, its presence implies Kmtracing == $true
so KMtracing could be removed

Reply:
The tracing and trace type parameters are now mandatory.